dealroom_firestore_connector/identifier.py

```diff
@@ -0,0 +1,82 @@
+from __future__ import annotations
+from typing import Union, Optional
+from .helpers import is_valid_uuid, is_valid_id
+from .exceptions import InvalidIdentifier
+
+
+# We mark the deleted entities from DR database with -2 entity so we can easily identify them.
+DELETED_DEALROOM_ENTITY_ID = -2
+
+# We mark the entities that don't exist in DR with -1.
+NOT_IN_DEALROOM_ENTITY_ID = -1
```

Comment:
Maybe we can also use a class like `Code` if that makes sense here (possible Dealroom Ids) and add the `is_valid_id` there

Reply:
Changed in https://github.com/dealroom/data-firestore-connector/pull/31/commits/e400b92cbe14e2cbe1509fef5839cabd7728530a